scss/spinners/10-fading-circle.scss

```diff
@@ -37,36 +40,20 @@
     content: '';
     display: block;
     margin: 0 auto;
-    width: 18%;
-    height: 18%;
+    width: 15%;
+    height: 15%;
     background-color: $spinkit-spinner-color;
     border-radius: 100%;
-    animation: sk-circleFadeDelay 1.2s infinite ease-in-out both;
+    animation: sk-circleFadeDelay $animationDuration infinite ease-in-out both;
   }
 
-  .sk-circle2  { transform: rotate(30deg) }
-  .sk-circle3  { transform: rotate(60deg) }
-  .sk-circle4  { transform: rotate(90deg) }
-  .sk-circle5  { transform: rotate(120deg) }
-  .sk-circle6  { transform: rotate(150deg) }
-  .sk-circle7  { transform: rotate(180deg) }
-  .sk-circle8  { transform: rotate(210deg) }
-  .sk-circle9  { transform: rotate(240deg) }
-  .sk-circle10 { transform: rotate(270deg) }
-  .sk-circle11 { transform: rotate(300deg) }
-  .sk-circle12 { transform: rotate(330deg) }
+  @for $i from 2 through $circleCount {
+    .sk-circle#{$i} { transform: rotate(360deg / $circleCount * ($i - 1)); }
+  }
 
-  .sk-circle2:before  { animation-delay: -1.1s }
-  .sk-circle3:before  { animation-delay: -1.0s }
-  .sk-circle4:before  { animation-delay: -0.9s }
-  .sk-circle5:before  { animation-delay: -0.8s }
-  .sk-circle6:before  { animation-delay: -0.7s }
-  .sk-circle7:before  { animation-delay: -0.6s }
-  .sk-circle8:before  { animation-delay: -0.5s }
-  .sk-circle9:before  { animation-delay: -0.4s }
-  .sk-circle10:before { animation-delay: -0.3s }
-  .sk-circle11:before { animation-delay: -0.2s }
-  .sk-circle12:before { animation-delay: -0.1s }
+  @for $i from 2 through $circleCount {
+    .sk-circle#{$i}:before { animation-delay: - $animationDuration + $animationDuration / $circleCount * ($i - 1); }
+  }
```

Comment:
> Why not let the HTML continue to be generated by the SCSS comments? We can create your demo pages in the gulpfile instead of the example HTML.

I'm all for continuing to let the HTML be generated by the SCSS comments, but I'm not talking about the challenge of fitting in the demo here. This is really about _usage_ by anyone who wants to customize the spinners. If you want to change the amount of spinners (after the changes in this branch), you can do so easily in the SCSS, but the HTML won't adapt. As a user, that kinda sucks :cry: